source/usr/lib/mcvirt/cluster/remote.py

```diff
@@ -367,7 +367,14 @@ def __connect(self):
             except AuthenticationException:
                 raise NodeAuthenticationException('Could not authenticate to node: %s' % self.name)
             except Exception:
-                raise CouldNotConnectToNodeException('Could not connect to node: %s' % self.name)
+                from mcvirt.auth import Auth
+
+                if (Auth().checkPermission(Auth.PERMISSIONS.CAN_IGNORE_CLUSTER)):
```

Comment:
The exception is always displayed, explaining that MCVirt could not connect to the remote node. But if the user can use the --ignore-cluster argument, then the exception will state that the user can use it to ignore the error.